doc/man/scc-hypervisor-collector.1.md

```diff
@@ -0,0 +1,216 @@
+exit immediately if it detects that it is running with superuser
+privileges.
+
+## CONFIGURATION SETTINGS & LOG FILE PERMISSIONS
+As the **scc-hypervisor-collector(5)** configuration settings
+will contain sensitive information such as passwords, the command
+requires that all specified configuration files and directories
+must be owned by the non-root user that is running the command,
+with restrictive permissions allowing only that user to access
+those files.
+
+Similarly, while every effort has been taken to ensure that no
+sensitive data is being written to the log files, to limit
+potential exposure of such information the log files must also
+be owned by, and only accessible by, the user that is running
+the **scc-hypervisor-collector(5)** command.
+
+## TLS/SSL CERTIFICATES
+
+The **virtual-host-gatherer(1)** framework doesn't currently
+permit the specification of certs that can be used to validate
+connections to the specified hypervisors.
+
+As such any certs that may be needed to securely connect to the
+specified hypervisors will need to be registered with the system
+certificate stores. See **update-ca-certificates(8)** for details.
+
+# SYSTEMD INTEGRATION
+
+The **scc-hypervisor-collector** package, when installed, automatically
+creates the **scchvc** restricted service account, if it doesn't already
+exist, with a home directory of **/var/lib/scchvc**.
+
+Additionally two **systemd(1)** units are also installed:
+
+* **scc-hypervisor-collector.service(8)** runs the **scc-hypervisor-collector**
+  as the **scchvc** user if a valid configuration has been specified in
+  the **scchvc** user account.
+
+* **scc-hypervisor-collector.timer(8)** triggers the
+  **scc-hypervisor-collector.service(8)** unit to be run on a daily
+  basis by default.
+
+# CONFIGURATION SETTINGS
+
+Configuration settings are specified in YAML format and must contain:
+
+* a list of hypervisor **backends** from which to retrieve relevant
+  details
+* a collection of **credentials** that will be used to upload the
+  collected details to the SUSE Customer Center.
+
+See **scc-hypervisor-collector(5)** for details about the possible
+configuration settings.
+
+## CONFIGURATION MANAGEMENT
+The configuration settings can be specified in a single YAML file,
+or as a set of YAML files whose content will be merged together in
+lexical sort order.
+
+The latter scheme allows the configuration settings to be split up
+into multiple files, e.g. credentials can be specified in one file,
+and the hypervisor backends in one or more files.
+
+When splitting the hypervisor backend details among multiple files,
+the **backends** lists in each file will be merged together to form
+a single combined list; exact duplicates will be ignored but partial
+duplicates will result in errors.
+
+## ACCESS AND OWNERSHIP
+For security reasons only the non-root user that is running the
+**scc-hypervisor-collector** command should be able to access the
```

Comment:
change non-root to:
scc-hypervisor-collector restricted service account (**scchvc**)

Reply:
Per our discussion have moved scchvc references to the scc-hypervisor-collect.service.8.md